### DISCUSSION TITLE
Support Data driven motion import

### DISCUSSION THREAD

**Discussion by GeorgeS2019** [GuilhermeGSousa/godot-motion-matching#32] [Ideas]
![image](https://github.com/user-attachments/assets/83540354-dffc-493e-9f9b-d761e0ce5445)

source: https://github.com/JLPM22/MotionMatching

**Comment by GuilhermeGSousa** [comment]
We already do this. During the animation baking process we extract features into the motion data array that we use for searching. I think the main difference is that we don't differentiate between "pose", "trajectory", etc, they're all considered features.
